src/main/java/calculator/domain/StringCalculator.java

```diff
@@ -12,11 +13,11 @@ public StringCalculator(Expression expression) {
     }
 
     public int calculate() {
-        Queue<String> operators = expression.getOperators();
+        Queue<Operator> operators = expression.getOperators();
         List<Integer> operands = expression.getOperands();
 
         return operands.stream()
-                .reduce((acc, curr) -> Operator.calculate(operators.poll(), acc, curr))
+                .reduce((acc, curr) -> Operator.calculate(Objects.requireNonNull(operators.poll()), acc, curr))
```

Comment:
String에서 Operator로 변경되었으니 calculate 메서드를 호출해도 될 것 같아요
```suggestion
                .reduce((acc, curr) -> (operators.poll().calculate(acc, curr)))
```